public/app/features/dashboard/components/SubMenu/DashboardLinksDashboard.tsx

```diff
@@ -68,16 +81,17 @@ export const DashboardLinksDashboard: React.FC<Props> = (props) => {
             <LinkElement
               link={link}
               key={`dashlinks-list-item-${resolvedLink.id}-${index}`}
-              aria-label={selectors.components.DashboardLinks.container}
+              data-testid={selectors.components.DashboardLinks.container}
             >
               <a
                 className="gf-form-label gf-form-label--dashlink"
                 href={resolvedLink.url}
                 target={link.targetBlank ? '_blank' : undefined}
                 rel="noreferrer"
-                aria-label={selectors.components.DashboardLinks.link}
+                data-testid={selectors.components.DashboardLinks.link}
```

Comment:
I imagine that the intention of having the data-testid instead of the aria-label is to use it for the unit test right? do we need to update the unit test somewhere by doing this change?